### PR TITLE
fix(stdlib): Avoid WASI random_get in Hash stdlib during module startup

### PR DESCRIPTION
**Pull request by ospencer**

### CODE REVIEW

File: stdlib/hash.gr [hunk 1]
```diff
@@ -247,7 +253,11 @@ let rec hashOne = (val, depth) => {
  */
```
Comment (spotandjake) [inline stdlib/hash.gr:253]: We should add the exception that `Result.unwrap` would throw to the grain doc to make sure it's known about.

**Review by spotandjake — CHANGES_REQUESTED** [review]
Looks great, I think we need to add some the exception that could occur to grain doc but otherwise lets get this merged.

**Comment by ospencer** [comment]
@spotandjake done.

**Review by spotandjake — APPROVED** [review]
Looks great.

**Comment by spotandjake** [comment]
It does look like before this gets merged the grain doc is going to need to be regenerated.

**Review by marcusroberts — APPROVED** [review]
Looks good to me!

**Pull request merged**